Assets/Scripts/New Features/QuestGiver.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Assertions;
+using TMPro;
+using System.Linq;
+
+public class QuestGiver : Interactable
+{
+    private const string progressFormat = "{0}/{1}";
+
+    [SerializeField]
+    private Canvas canvas = null;
+    //each quest giver needs a list of quests
+    //need to open a panel that loads those quests
```

Comment:
What is this a comment for? Put comments directly above what they are referring to